sections/currentState.tex

```diff
@@ -0,0 +1,31 @@
+\section{Current state}
+
+As previously mentioned, \knox{} was initially started in 2020.
+The previous group chose to develop this layer using Java.
+
+Using the framework Apache Jena in combination with the Fuseki package, they set up a simple database system.
+
+The system has two functionalities. 
+Firstly, it can store knowledge graphs in HDT and convert triples to HDT.
+Secondly, it contains a prototype for counting the number of words in an article.
+There is no API for reading this data, only for writing. 
+In addition, the other layers are directly connected to the database.
+
+Overall, not much code was written and its quality is questionable. 
```

Comment:
Nickpick: Maybe this should be phrased different. We aren't documenting what has not been done, only what is done.